graylog2-web-interface/src/components/search/SidebarMessageField.jsx

```diff
@@ -18,6 +17,27 @@ const SidebarMessageField = React.createClass({
   _toggleShowActions() {
     this.setState({showActions: !this.state.showActions});
   },
+  _setShowActions(isOpen) {
+    this.setState({showActions: isOpen});
+  },
```

Comment:
In that case we can remove `_toggleShowActions`, as the toggle will be controller by the `DropdownButton`, and will update the state using `_setShowActions`.